helm/policy-api-crds/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+appVersion: 0.0.1
+name: policy-api-crds
+description: This app contains the Policy API CRDs.
+home: https://github.com/giantswarm/policy-api
+icon: https://s.giantswarm.io/app-icons/giantswarm/1/light.svg
+version: 0.0.0
```

Comment:
What are your thoughts on versioning?

Reply:
It will match `policy-api` release version **_I hope_**. I'm hoping the workflows will make it match, if not I will just set it to the same version.